lib/smart_answer_flows/benefit-cap-calculator.rb

```diff
@@ -8,6 +8,16 @@ def define
 
       config = Calculators::BenefitCapCalculatorConfiguration.new
 
+      # Routing question
+      multiple_choice :choose_cap_to_calculate? do
+        option :default
+        option :future
+
+        next_node do
+          question :receive_housing_benefit?
+        end
+      end
+
       # Q1
       multiple_choice :receive_housing_benefit? do
         option :yes
```

Comment:
> TODO: Consider adding only single response to the first question for now.

This TODO should be removed from the commit note.

This commit is quite big, so it might be worth trying the suggestion in the TODO.